randovania/games/prime/patcher_file_lib/item_hints.py

```diff
@@ -43,15 +43,13 @@
 ]
 
 _JOKE_HINTS = [
```

Comment:
We should leave the rules for a joke hint to be added as a comment here.

Reply:
Agreed. Are the ones I put in #403 okay?

Reply:
I think you meant the issue, not a circular link. But yeah.